contracts/VeJoeStaking.sol

```diff
@@ -41,6 +41,9 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
     /// For example, if user has `n` JOE staked, they can own a maximum of `n * maxCap` veJOE.
     uint256 public maxCap;
 
+    /// @notice The upper limit of `maxCap`
+    uint256 public MAX_MAX_CAP;
```

Comment:
I don’t think this is a good name for a variable..
Maybe `UPPER_BOUND_MAX_CAP` or `UPPER_LIMIT_MAX_CAP` ?